radix-engine/src/vm/native_vm.rs

```diff
@@ -147,7 +147,18 @@ impl<I: VmInvoke> VmInvoke for NativeVmInstance<I> {
                     ROLE_ASSIGNMENT_CODE_ID => {
                         RoleAssignmentNativePackage::invoke_export(export_name, input, api)
                     }
-                    POOL_CODE_ID => PoolNativePackage::invoke_export(export_name, input, api),
+                    POOL_V1_0_CODE_ID => PoolNativePackage::invoke_export(
+                        export_name,
+                        input,
+                        PoolV1MinorVersion::Zero,
+                        api,
+                    ),
+                    POOL_V1_1_CODE_ID => PoolNativePackage::invoke_export(
```

Comment:
Minor: I think this should be a separate `PoolNativePackageV1_1` rather than passing in a minor version as the `invoke_export` represents calling a new "code file"